features/attached_enable.feature

```diff
@@ -385,13 +390,60 @@ Feature: Enable command behaviour when attached to an UA subscription
             Installing canonical-livepatch snap
             Canonical livepatch enabled
             """
-        And I verify that running `ua enable fips --assume-yes` `with sudo` exits `1`
+        When I append the following on uaclient config:
+        """
+        features:
+          block_disable_on_enable: true
+        """
+        Then I verify that running `ua enable fips --assume-yes` `with sudo` exits `1`
         And I will see the following on stdout
             """
             One moment, checking your subscription first
             Cannot enable FIPS when Livepatch is enabled
             """
 
+    @series.xenial
+    @series.bionic
+    @uses.config.machine_type.lxd.vm
+    Scenario Outline: Attached enable fips on a machine with livepatch active
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I attach `contract_token` with sudo
+        Then stdout matches regexp:
+            """
+            Updating package lists
+            ESM Infra enabled
+            """
+        And stdout matches regexp:
+            """
+            Installing canonical-livepatch snap
+            Canonical livepatch enabled
+            """
+        When I run `ua enable fips --assume-yes --allow-disable` with sudo
```

Comment:
what is the difference in the aasume-yes vs the allow-disable?

Reply:
Maybe I am overthinking that, but my rationale here is that, by default, we should ask the users for confirmation on the disable operation. Even though the `--assume-yes`  already dictates that it will answer yes to all prompts that appear, it may not be clear to the user that during an enable operation we could also disable existing services.

Reply:
The reason for folks to use --assume-yes is for scripted where no one is there and they've added that to make sure it happens (e.g. enables FIPS) so I think just --assume-yes would be sufficient for users. 

I get the idea of "allow disabling of services" now as what you meant with allow-disable. I just don't think we need that separate permission.